packages/app/src/App.tsx

```diff
@@ -83,6 +84,7 @@ const AppRoutes = () => (
       element={<RegisterComponentRouter catalogRouteRef={catalogRouteRef} />}
     />
     <Route path="/settings" element={<SettingsRouter />} />
+    <Route path="/import-catalog/*" element={<CatalogImportRouter />} />
```

Comment:
I agree, but for some reason if the route is `catalog-*`, blank page loads instead of the pluginRouter component. That's why for now i've inverted this route. I think I will debug it at the end of the work.